spec/support/dashboard_helper.rb

```diff
@@ -1,38 +1,5 @@
-require_relative 'wait_for_ajax.rb'
-
 module DashboardHelper
-  extend ActiveSupport::Concern
-  include WaitForAjax
-
-  SHORT_DASHBOARD_REFRESH_SECONDS = 0.1
-
-  # Some integration tests run assertions dependant on refreshing the dashboard,
-  # this means there is an unavoidable sleep that needs to happen while we wait
-  # for the time interval to elapse to trigger a periodic refresh. If we need
-  # to do this, then we can temporarily set the refresh period to something
-  # shorter than standard.
-  #
-  # We only want to do this in tagged specs, reducing the refresh period for all
-  # tests will make things unstable if they are trying to interact with elements
-  # on the dashboard whilst it is refreshing.
-
-  included do
-    around(:example, short_dashboard_refresh: true) do |example|
-      old_refresh = Settings.dsds.dashboard_refresh_milliseconds
-      Settings.dsds.dashboard_refresh_milliseconds = short_dashboard_refresh_milliseconds
-      example.run
-      Settings.dsds.dashboard_refresh_milliseconds = old_refresh
-    end
-  end
-
-  def wait_for_dashboard_refresh
-    sleep(Settings.dsds.dashboard_refresh_milliseconds / 1000)
-    wait_for_ajax
-  end
-
-  private
-
-  def short_dashboard_refresh_milliseconds
-    SHORT_DASHBOARD_REFRESH_SECONDS * 1000
+  def refresh_dashboard
+    page.execute_script "window.jQuery.ajax({url: 'refresh_dashboard'})"
```

Comment:
this is bypassing the triggering of the dashboard refresh. The 'old' way being removed here is a better integration test as the dashboard refresh is being triggered by the same mechanism (just with a reduced wait time).

If we trigger the refresh explicitly like here we need a different test to test the actual triggering mechanism defined in app/assets/javascripts/dashboard_refresh_interval.js.erb